src/pybind/mgr/mgr_module.py

```diff
@@ -343,6 +343,67 @@ def _unit_to_str(self, unit):
         elif unit == self.BYTES:
             return "B/s"  
     
+    def to_pretty_iec(self, n):
+        bump_after = 10
+        if n > bump_after << 60:
```

Comment:
```py
for bits, suffix in {60: 'Ei', 50, 'Pi', 40: 'Ti',
                     30: 'Gi', 20: 'Mi', 10: 'Ki'}:
    if n > 10 << bits:
        return str(n << bits) + ' ' + suffix
return str(n) + ' '
```

Reply:
Updated with a minor change: sort the elements by key as there is no guarantee on the order of elements when iterating over a dictionary.